src/linux_mcp_server/commands.py

```diff
@@ -148,6 +148,42 @@ def get_command(name: str) -> CommandEntry:
     return COMMANDS[name]
 
 
+def get_command_spec(key: str) -> CommandSpec:
```

Comment:
This functionality can be removed and all the calls to it replaced by the using the [validate_call decorator](https://docs.pydantic.dev/latest/concepts/validation_decorator/). This lets Pydantic handle input and optionally return validation and coercion.